src/components/Modal/Modal.vue

```diff
@@ -637,7 +637,7 @@ $header-size: 50px;
 		align-items: center;
 		justify-content: center;
 		width: 15%;
-		min-width: 60px;
+		min-width: 35px;
```

Comment:
Min-width should be 44px, no? Otherwise looks good! :)

Reply:
Oh right, missed that! Indeed!
You can use `$clickable-area` @GretaD

Reply:
But I would rather not change this here please. Because this will affect all sizes.
Please ass dedicated `.prev, .next` children in the --large one